packages/hop-node/src/db/TransfersDb.ts

```diff
@@ -438,6 +462,82 @@ class TransfersDb extends BaseDb {
     return sorted as UnbondedSentTransfer[]
   }
 
+  async getUnrelayedSentTransfers (
+    filter: GetItemsFilter = {}
+  ): Promise<UnrelayedSentTransfer[]> {
+    // TODO: Remove this post-nitro
+    if (!isNitroLive) {
+      return []
+    }
+
+    const transfers: Transfer[] = await this.getTransfersFromWeek()
+    const filtered = transfers.filter(item => {
+      if (!item?.transferId) {
+        return false
+      }
+
+      if (!this.isRouteOk(filter, item)) {
+        return false
+      }
+
+      if (item.isNotFound) {
+        return false
+      }
+
+      if (item?.sourceChainSlug !== Chain.Ethereum) {
+        return false
+      }
+
+      if (!item?.destinationChainSlug) {
+        return false
+      }
+
+      if (!RelayableChains.includes(item.destinationChainSlug)) {
+        return false
+      }
+
+      // TODO: Remove this one week post-nitro
+      if (item.transferSentTimestamp && item.transferSentTimestamp < nitroStartTimestamp) {
+        return false
+      }
+
+      let timestampOk = true
+      if (item.relayAttemptedAt) {
+        if (TxError.RelayerFeeTooLow === item.relayTxError) {
+          const delay = TxRetryDelayMs + ((1 << item.relayBackoffIndex!) * 60 * 1000) // eslint-disable-line
+          // TODO: use `sentTransferTimestamp` once it's added to db
+
+          // don't attempt to relay after a week
+          if (delay > OneWeekMs) {
+            return false
+          }
+          timestampOk = item.relayAttemptedAt + delay < Date.now()
+        } else {
+          timestampOk = item.relayAttemptedAt + TxRetryDelayMs < Date.now()
+        }
+      }
+
+      const seenOnL1TimestampMs: number = item.transferSentTimestamp! * 1000
```

Comment:
I think it should return early if `transferSentTimestamp` is undefined (incomplete item)

```ts
if (!item.transferSentTimestamp) {
  return false
}
```